tutorials/connecting-vaadin-applications-to-mysql-databases/content.adoc

```diff
@@ -0,0 +1,328 @@
+= Connecting Vaadin Applications to MySQL Databases
+
+:type: text
+:tags: Databases,SQL
+:description: Learn how connect Vaadin applications to MySQL databases.
+:repo:
+:linkattrs:
+:imagesdir: ./images
+:related_tutorials:
+
+In this tutorial, you'll learn how to connect to a MySQL database to save, update, and delete data from a Vaadin web application using the Java Persistence API (JPA).
```

Comment:
We should guide towards using JPA via Java EE or Spring Data JPA as primary solution.